### ISSUE TITLE
Turn off drag and drop for specific item node or group of nodes.

### ISSUE THREAD

**Issue by WriterStat** [mbraak/jqTree#591]
Hi Marco, thanks for the great software!

Is there a way to turn off a specific node's drag and drop? Or group of nodes?

Example: I have top level nodes that shouldn't be dragged.

Example 2: I have nodes inside of the top level nodes that shouldn't be dragged/dropped. But some of the nodes below them can be dragged/dropped.

Something like sticky nodes? Static nodes. Sort of like a sticky note in a forum that always stays in place on the top?

Thanks!

**Comment by mbraak** [comment]
I think you can use the [onCanMove](https://mbraak.github.io/jqTree/#options-oncanmove) option.

**Comment by WriterStat** [comment]
Thanks! Sorry I must have missed this. Not enough caffeine. =) 

Should definitely do it. I could set a json flag/json group flag on the node, no_drag and then check it here.

**Comment by WriterStat** [comment]
Is there a way to

**Comment by WriterStat** [comment]
Hi, thanks!

On 'onCanMoveTo' what's the best way to keep a moved_node from moving to the root/top level?

thanks!

**Comment by WriterStat** [comment]
On 'onCanMoveTo' what's the best way to keep a moved_node from moving inside a target_node?

position != 'inside'

thanks!

Thanks again!

Almost there. =)

**Comment by WriterStat** [comment]
Or On 'onCanMoveTo' what's the best way to allow a moved_node to move both before and after a target_node? But not inside.

Thanks!

**Comment by WriterStat** [comment]
ok, I think I got it. =) Thanks! And thanks for the great software!

I guessed a bit and used:

```
return (position == 'before' || position == 'after');
```
I can now turn off a specific node's drag and drop. And turn off a group of node's drag and drop. And limit which nodes can be dragged/dropped in a group of nodes and where inside, those nodes can be dragged/dropped.

I can also set and limit which nodes from where can be dragged and dropped inside which top nodes.

I can now set it to: Top level nodes shouldn't be dragged/dropped.

I can now have sticky nodes that will stay at the top inside a node. Or a group of nodes that are sticky/stay at the top inside a node. While still allowing the other nodes to be moved inside that node while setting limits or to another top node that allows drag/drop.

Thanks again!
